website/public/js/filters/filters.js

```diff
@@ -21,3 +21,11 @@ angular.module('habitrpg')
         (gp > 999) ? (gp / Math.pow(10, 3)).toFixed(1) + "k" : gp;
     }
   })
+  .filter('conditionalOrderBy', function($filter) {
```

Comment:
this isn't minification ready, you should use the array notation `['$filter', function($filter){}]` see https://github.com/HabitRPG/habitrpg/blob/b503a0fd641b7dc9f031b65c8dcd7f80be7b81bd/website/public/js/services/challengeServices.js#L8 for an example